frontend/public/components/app.jsx

```diff
@@ -129,8 +130,8 @@ class App extends React.PureComponent {
         <GlobalNotifications />
         <Switch>
           <Route path={['/all-namespaces', '/ns/:ns',]} component={RedirectComponent} />
-          <LazyRoute path="/status/all-namespaces" exact loader={() => import('./cluster-overview' /* webpackChunkName: "cluster-overview" */).then(m => m.ClusterOverviewPage)} />
-          <LazyRoute path="/status/ns/:ns" exact loader={() => import('./cluster-overview' /* webpackChunkName: "cluster-overview" */).then(m => m.ClusterOverviewPage)} />
+          <Route path="/status/all-namespaces" exact component={ClusterOverviewPage} />
+          <Route path="/status/ns/:ns" exact component={ClusterOverviewPage} />
```

Comment:
You should be able to revert these changes now

Reply:
D'oh.  Thanks.